SS14.Launcher/ViewModels/MainWindowTabs/ServerEntryViewModel.cs

```diff
@@ -72,9 +72,7 @@ public void ConnectPressed()
     public string ServerStatusString => _cacheData.Status switch
     {
         ServerStatusCode.Offline => "Unable to connect",
-        ServerStatusCode.Online => _cacheData.PlayerCount == 1
-            ? $"Online: {_cacheData.PlayerCount} player"
-            : $"Online: {_cacheData.PlayerCount} players",
+        ServerStatusCode.Online => $"Online: {_cacheData.PlayerCount} / {_cacheData.SoftMaxPlayerCount} players",
```

Comment:
this needs to default to the old message in the case of softmaxplayers being undefined (i.e. defaulted to 0)

Reply:
could happen if a server isn't updated, if they removed soft max players or adding it to the json, etc

Reply:
Thanks for the clarification, I was wondering if it was possible to be undefined since it seemed required for servers right now